packages/lib-classifier/src/store/TranscriptionReductions/TranscriptionReductions.js

```diff
@@ -0,0 +1,125 @@
+import asyncStates from '@zooniverse/async-states'
+import { flow, getEnv, types } from 'mobx-state-tree'
+
+const CONSENSUS_SCORE_TO_RETIRE = 3
+const MINIMUM_VIEW_TO_RETIRE = 5
+const REDUCER_KEY = 'alice'
+
+const TranscriptionLine = types.model({
```

Comment:
The reduced lines are in the reductions array already. I’ve used `TranscribedLines` elsewhere so `TranscribedLine`?

Reply:
I'd prefer the more explicit naming since `TranscribedLine` could be thought of as another name for an annotation. The model should have a node name too, `types.model('ReducedTranscriptionLine', {`

Reply:
It isn’t a reduced line though. That’s my only problem with the name.

Reply:
I've deleted this model.